### ISSUE TITLE
Generate boilerplate for quick-start

### ISSUE THREAD

**Issue by aitikgupta** [Open-Sourced-Olaf/vscode-github-projects#1]
Quickstart from official documentation to bootstrap the extension in TypeScript.
(Also to test how issue-integration with Projects work)

**Comment by ShrillShrestha** [comment]
I tried running it locally and everything looks fine so far. But I might want to suggest doing:
```
context.subscriptions.push(vscode.commands.registerCommand('vscode-github-projects.helloWorld', () => {
  // The code you place here will be executed every time your command is executed
  
  // Display a message box to the user
  vscode.window.showInformationMessage('Hello World from vscode-github-projects!');
  })
);
```
instead of making a `disposable` variable and then pushing it into `context.subscriptions`. It looks more concise.
What do you think @aitikgupta, @gladishd?

**Comment by gladishd** [comment]
@ShrillShrestha you know what?  I think that would be incredible.  I have never put a disposable variable into context.subscriptions before this project.  Before this week, I used to think life was a linear game.  I treated my code the same way, although now I think I can see the beauty of reading things right to left as well as left to right.  Shrill, this is an excellent convention which we should implement (right?) and I will be sure to do this next time I work on the project some time tomorrow, that's how we do things around here.
